mirgecom/euler.py

```diff
@@ -57,16 +57,13 @@
     inviscid_flux,
     inviscid_facial_flux
 )
-from grudge.eager import (
-    interior_trace_pair,
-    cross_rank_trace_pairs
-)
-from grudge.trace_pair import TracePair
-from mirgecom.fluid import make_conserved
+
+from grudge.trace_pair import interior_trace_pairs
 from mirgecom.operators import div_operator
 
 
-def euler_operator(discr, eos, boundaries, cv, time=0.0):
+def euler_operator(discr, eos, boundaries, cv, time=0.0,
+                   dv=None):
```

Comment:
@MTCam Where did this (`dv=None`) come from? Is this a leftover from https://github.com/illinois-ceesd/mirgecom/pull/566/commits/9e8824ac1157fda9bbd4212a68c932b4626c3414?

Reply:
Cruft removed (6e2e4d3)